packages/addon/public/manifest.json

```diff
@@ -38,6 +38,22 @@
     "webRequestBlocking",
     "messagesRead",
     "sensitiveDataUpload",
-    "https://*.backblazeb2.com/*"
+    "https://*.backblazeb2.com/*",
+    "tabs",
+    "activeTab",
+    "http://localhost:5173/*"
+  ],
+  "browser_action": {
+    "default_title": "TB Pro",
+    "default_icon": "icons/64-dev.png",
+    "allowed_spaces": [],
+    "default_popup": "tbpro-menu.html"
+  },
+  "content_scripts": [
+    {
+      "matches": ["<all_urls>"],
```

Comment:
This is a big security no-no. The real URLs should be used here for the web app and the auth server.